src/gluonts/ev/helpers.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import Dict, Optional
+import numpy as np
+
+
+EvalData = Dict[str, np.ndarray]
+
+
+def axis_is_zero_or_none(axis: Optional[int]) -> bool:
```

Comment:
I don't this there is a need for this. Let's just use `axis is Nore or axis == 0` directly instead.